frontend/lib/components/TextEditor/TextEditor.module.scss

```diff
@@ -0,0 +1,129 @@
+@use "styles/Spacings.module.scss";
+@use "styles/Typography.module.scss";
+@use "styles/Radius.module.scss";
+@use "styles/ScreenSizes.module.scss";
+
+.search_bar_wrapper {
+  position: sticky;
+  bottom: Spacings.$spacing06;
+  // width: 50%;
+}
+
+.editor_wrapper {
+  max-width: ScreenSizes.$medium;
+  width: 100%;
+  margin: 0 auto;
+  padding: Spacings.$spacing07;
+  min-height: 100vh;
+
+  .content_wrapper {
+    color: var(--text-3);
+    width: 100%;
+    position: relative;
+
+    .has_focus {
+      position: relative;
+
+      &::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        right: calc(100% + Spacings.$spacing04);
+        width: Spacings.$spacing02;
+        background-color: var(--primary-0);
+        height: 100%;
+      }
+    }
+
+    h1 {
+      @include Typography.Big;
+      font-weight: 700;
+    }
+
+    h2 {
+      @include Typography.H1;
+    }
+
+    h3 {
+      @include Typography.H2;
+    }
```

Comment:
A bit weird, not your fault
Could you change in all scss files this ? H1 should be big, H2 H1, etc